2019-07-02 12:20:21.961582: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2019-07-02 12:20:23.089814: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1405] Found device 0 with properties: 
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.582
pciBusID: 0000:03:00.0
totalMemory: 10.92GiB freeMemory: 10.76GiB
2019-07-02 12:20:23.090120: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
2019-07-02 12:20:23.090166: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-07-02 12:20:23.090190: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
2019-07-02 12:20:23.090198: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
2019-07-02 12:20:23.093113: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
2019-07-02 12:20:23.489356: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-07-02 12:20:23.489408: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
2019-07-02 12:20:23.489416: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
2019-07-02 12:20:23.489887: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/device:GPU:0 with 10405 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:03:00.0, compute capability: 6.1)
ERROR: cannot read default config file. using build-in defaults [main]
LOG: connected to ('10.6.100.1', 10321)
LOG: destination: /var/tmp/makarov/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: request: /work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-222:/var/tmp/zeineldeen/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-222:/var/tmp/zeineldeen/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-272:/var/tmp/bahar/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-272:/var/tmp/bahar/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-220:/var/tmp/zeyer/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-220:/var/tmp/zeyer/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-247:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-247:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-256:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-256:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-262:/var/tmp/makarov/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-262:/var/tmp/makarov/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-231:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-231:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-228:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-228:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-280:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-280:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-242:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-242:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-235:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-235:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-248:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-248:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-229:/var/tmp/bahar/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-229:/var/tmp/bahar/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-225:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-225:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-240:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-240:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-216:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-216:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-230:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-230:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-224:/var/tmp/zeyer/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-224:/var/tmp/zeyer/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-263:/var/tmp/bahar/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-263:/var/tmp/bahar/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-237:/var/tmp/zeineldeen/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-237:/var/tmp/zeineldeen/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-269:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-269:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-238:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-238:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-264:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-264:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-217:/var/tmp/bahar/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-217:/var/tmp/bahar/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-02:/var/tmp/zeineldeen/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: start copying cluster-cn-02:/var/tmp/zeineldeen/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: copied cluster-cn-02:/var/tmp/zeineldeen/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
2019-07-02 12:20:39.567596: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
2019-07-02 12:20:39.567683: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-07-02 12:20:39.567697: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
2019-07-02 12:20:39.567706: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
2019-07-02 12:20:39.567960: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10405 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:03:00.0, compute capability: 6.1)
2019-07-02 12:21:59.209419: W tensorflow/core/framework/op_kernel.cc:1275] OP_REQUIRES failed at save_restore_v2_ops.cc:184 : Permission denied: /work/smt3/bahar/expriments/asr/librispeech/test/20190121-end2end-hmm/data-train/trafo.specaug.datarndperm_noscale.12l.ffdim4.pretrain3-prior-k6/net-model/network.560.index; Permission denied
[31;1mEXCEPTION[0m
[34mTraceback (most recent call last):[0m
  [34;1mFile [0m[36m"/u/makarov/returnn-parnia/tools/[0m[36;1mget-attention-weights.py[0m[36m"[0m, [34mline [0m[35m414[0m, [34min [0m<module>
[34m    line: [0mmain[34m([0msys[34m.[0margv[34m)[0m
[34m    locals:[0m
      main [34;1m= [0m[34m<local> [0m[34m<[0mfunction main at 0x2b784230c730[34m>[0m
      sys [34;1m= [0m[34m<local> [0m[34m<[0mmodule [36m'sys'[0m [34m([0mbuilt[34m-[0m[34min[0m[34m)[0m[34m>[0m
      sys[34;1m.[0margv [34;1m= [0m[34m<local> [0m[34m[[0m[36m'/u/makarov/returnn-parnia/tools/get-attention-weights.py'[0m[34m,[0m [36m'/u/bahar/workspace/asr/librispeech/test-20190121/config-train/trafo.specaug.datarndperm_noscale.12l.ffdim4.pretrain3-prior-k6.config'[0m[34m,[0m [36m'--epoch'[0m[34m,[0m [36m'560'[0m[34m,[0m [36m'--data'[0m[34m,[0m [36m'config:get_dataset("dev-clean")'[0m[34m,[0m [36m'--dump_dir'[0m[34m,[0m [36m'/u/makarov/makarov/NMT..., len = 40, _[0]: {len = 56}[0m
  [34;1mFile [0m[36m"/u/makarov/returnn-parnia/tools/[0m[36;1mget-attention-weights.py[0m[36m"[0m, [34mline [0m[35m231[0m, [34min [0mmain
[34m    line: [0minit_net[34m([0margs[34m,[0m layers[34m)[0m
[34m    locals:[0m
      init_net [34;1m= [0m[34m<global> [0m[34m<[0mfunction init_net at 0x2b784230c6a8[34m>[0m
      args [34;1m= [0m[34m<local> [0mNamespace[34m([0masr[34m=[0m[34mFalse[0m[34m,[0m batch_size[34m=[0m4000[34m,[0m beam_size[34m=[0m12[34m,[0m config_file[34m=[0m[36m'/u/bahar/workspace/asr/librispeech/test-20190121/config-train/trafo.specaug.datarndperm_noscale.12l.ffdim4.pretrain3-prior-k6.config'[0m[34m,[0m data[34m=[0m[36m'config:get_dataset("dev-clean")'[0m[34m,[0m device[34m=[0m[34mNone[0m[34m,[0m do_search[34m=[0m[34mFalse[0m[34m,[0m dropout[34m=[0m[34mNone[0m[34m,[0m dump_dir[34m=[0m[36m'/u/...[0m
      layers [34;1m= [0m[34m<local> [0m[34m[[0m[36m'dec_02_att_weights'[0m[34m,[0m [36m'dec_01_att_weights'[0m[34m,[0m [36m'dec_03_att_weights'[0m[34m,[0m [36m'dec_04_att_weights'[0m[34m,[0m [36m'dec_05_att_weights'[0m[34m,[0m [36m'dec_06_att_weights'[0m[34m,[0m [36m'dec_07_att_weights'[0m[34m,[0m [36m'dec_08_att_weights'[0m[34m,[0m [36m'dec_09_att_weights'[0m[34m,[0m [36m'dec_10_att_weights'[0m[34m,[0m [36m'dec_11_att_weights'[0m[34m,[0m [36m'dec_12_att_weights'[0m[34m][0m[34m,[0m len [34m=[0m 12[34m,[0m _[34m[[0m0[34m][0m[34m:[0m [34m{[0mlen [34m=[0m 18[34m}[0m
  [34;1mFile [0m[36m"/u/makarov/returnn-parnia/tools/[0m[36;1mget-attention-weights.py[0m[36m"[0m, [34mline [0m[35m139[0m, [34min [0minit_net
[34m    line: [0mrnn[34m.[0mengine[34m.[0minit_network_from_config[34m([0mconfig[34m=[0mconfig[34m,[0m net_dict_post_proc[34m=[0mnet_dict_post_proc[34m)[0m
[34m    locals:[0m
      rnn [34;1m= [0m[34m<global> [0m[34m<[0mmodule [36m'rnn'[0m [34mfrom [0m[36m'/u/makarov/returnn-parnia/rnn.py'[0m[34m>[0m
      rnn[34;1m.[0mengine [34;1m= [0m[34m<global> [0m[34m<[0mTFEngine[34m.[0mEngine object at 0x2b779327fac8[34m>[0m
      rnn[34;1m.[0mengine[34;1m.[0minit_network_from_config [34;1m= [0m[34m<global> [0m[34m<[0mbound method Engine[34m.[0minit_network_from_config of [34m<[0mTFEngine[34m.[0mEngine object at 0x2b779327fac8[34m>[0m[34m>[0m
      config [34;1m= [0m[34m<global> [0m[34m<[0mConfig[34m.[0mConfig object at 0x2b784230f668[34m>[0m
      net_dict_post_proc [34;1m= [0m[34m<local> [0m[34m<[0mfunction init_net[34m.[0m[34m<[0mlocals[34m>[0m[34m.[0mnet_dict_post_proc at 0x2b78b8d28ae8[34m>[0m
  [34;1mFile [0m[36m"/u/makarov/returnn-parnia/[0m[36;1mTFEngine.py[0m[36m"[0m, [34mline [0m[35m967[0m, [34min [0minit_network_from_config
[34m    line: [0mself[34m.[0mnetwork[34m.[0mload_params_from_file[34m([0mmodel_epoch_filename[34m,[0m session[34m=[0mself[34m.[0mtf_session[34m)[0m
[34m    locals:[0m
      self [34;1m= [0m[34m<local> [0m[34m<[0mTFEngine[34m.[0mEngine object at 0x2b779327fac8[34m>[0m
      self[34;1m.[0mnetwork [34;1m= [0m[34m<local> [0m[34m<[0mTFNetwork [36m'root'[0m train[34m=[0m[34m<[0mtf[34m.[0mTensor [36m'globals/train_flag:0'[0m shape[34m=[0m[34m([0m[34m)[0m dtype[34m=[0mbool[34m>[0m[34m>[0m
      self[34;1m.[0mnetwork[34;1m.[0mload_params_from_file [34;1m= [0m[34m<local> [0m[34m<[0mbound method TFNetwork[34m.[0mload_params_from_file of [34m<[0mTFNetwork [36m'root'[0m train[34m=[0m[34m<[0mtf[34m.[0mTensor [36m'globals/train_flag:0'[0m shape[34m=[0m[34m([0m[34m)[0m dtype[34m=[0mbool[34m>[0m[34m>[0m[34m>[0m
      model_epoch_filename [34;1m= [0m[34m<local> [0m[36m'/work/smt3/bahar/expriments/asr/librispeech/test/20190121-end2end-hmm/data-train/trafo.specaug.datarndperm_noscale.12l.ffdim4.pretrain3-prior-k6/net-model/network.560'[0m[34m,[0m len [34m=[0m 166
      session [34;1m= [0m[34m<not found>[0m
      self[34;1m.[0mtf_session [34;1m= [0m[34m<local> [0m[34m<[0mtensorflow[34m.[0mpython[34m.[0mclient[34m.[0msession[34m.[0mSession object at 0x2b77e8e15cc0[34m>[0m
  [34;1mFile [0m[36m"/u/makarov/returnn-parnia/[0m[36;1mTFNetwork.py[0m[36m"[0m, [34mline [0m[35m954[0m, [34min [0mload_params_from_file
[34m    line: [0mself[34m.[0msaver[34m.[0mrestore[34m([0msess[34m=[0msession[34m,[0m save_path[34m=[0mfilename[34m)[0m
[34m    locals:[0m
      self [34;1m= [0m[34m<local> [0m[34m<[0mTFNetwork [36m'root'[0m train[34m=[0m[34m<[0mtf[34m.[0mTensor [36m'globals/train_flag:0'[0m shape[34m=[0m[34m([0m[34m)[0m dtype[34m=[0mbool[34m>[0m[34m>[0m
      self[34;1m.[0msaver [34;1m= [0m[34m<local> [0m[34m<[0mtensorflow[34m.[0mpython[34m.[0mtraining[34m.[0msaver[34m.[0mSaver object at 0x2b7c29f4fc18[34m>[0m
      self[34;1m.[0msaver[34;1m.[0mrestore [34;1m= [0m[34m<local> [0m[34m<[0mbound method Saver[34m.[0mrestore of [34m<[0mtensorflow[34m.[0mpython[34m.[0mtraining[34m.[0msaver[34m.[0mSaver object at 0x2b7c29f4fc18[34m>[0m[34m>[0m
      sess [34;1m= [0m[34m<not found>[0m
      session [34;1m= [0m[34m<local> [0m[34m<[0mtensorflow[34m.[0mpython[34m.[0mclient[34m.[0msession[34m.[0mSession object at 0x2b77e8e15cc0[34m>[0m
      save_path [34;1m= [0m[34m<not found>[0m
      filename [34;1m= [0m[34m<local> [0m[36m'/work/smt3/bahar/expriments/asr/librispeech/test/20190121-end2end-hmm/data-train/trafo.specaug.datarndperm_noscale.12l.ffdim4.pretrain3-prior-k6/net-model/network.560'[0m[34m,[0m len [34m=[0m 166
  [34;1mFile [0m[36m"/u/bahar/settings/python3-returnn-tf1.9/lib/python3.5/site-packages/tensorflow/python/training/[0m[36;1msaver.py[0m[36m"[0m, [34mline [0m[35m1725[0m, [34min [0mrestore
[34m    line: [0msess[34m.[0mrun[34m([0mself[34m.[0msaver_def[34m.[0mrestore_op_name[34m,[0m
                   [34m{[0mself[34m.[0msaver_def[34m.[0mfilename_tensor_name[34m:[0m save_path[34m}[0m[34m)[0m
[34m    locals:[0m
      sess [34;1m= [0m[34m<local> [0m[34m<[0mtensorflow[34m.[0mpython[34m.[0mclient[34m.[0msession[34m.[0mSession object at 0x2b77e8e15cc0[34m>[0m
      sess[34;1m.[0mrun [34;1m= [0m[34m<local> [0m[34m<[0mbound method BaseSession[34m.[0mrun of [34m<[0mtensorflow[34m.[0mpython[34m.[0mclient[34m.[0msession[34m.[0mSession object at 0x2b77e8e15cc0[34m>[0m[34m>[0m
      self [34;1m= [0m[34m<local> [0m[34m<[0mtensorflow[34m.[0mpython[34m.[0mtraining[34m.[0msaver[34m.[0mSaver object at 0x2b7c29f4fc18[34m>[0m
      self[34;1m.[0msaver_def [34;1m= [0m[34m<local> [0mfilename_tensor_name[34m:[0m [36m"saver/save/Const:0"[0m
                               save_tensor_name[34m:[0m [36m"saver/save/control_dependency:0"[0m
                               restore_op_name[34m:[0m [36m"saver/save/restore_all"[0m
                               max_to_keep[34m:[0m 2147483647
                               keep_checkpoint_every_n_hours[34m:[0m 10000[34m.[0m0
                               version[34m:[0m V2

      self[34;1m.[0msaver_def[34;1m.[0mrestore_op_name [34;1m= [0m[34m<local> [0m[36m'saver/save/restore_all'[0m[34m,[0m len [34m=[0m 22
      self[34;1m.[0msaver_def[34;1m.[0mfilename_tensor_name [34;1m= [0m[34m<local> [0m[36m'saver/save/Const:0'[0m[34m,[0m len [34m=[0m 18
      save_path [34;1m= [0m[34m<local> [0m[36m'/work/smt3/bahar/expriments/asr/librispeech/test/20190121-end2end-hmm/data-train/trafo.specaug.datarndperm_noscale.12l.ffdim4.pretrain3-prior-k6/net-model/network.560'[0m[34m,[0m len [34m=[0m 166
  [34;1mFile [0m[36m"/u/bahar/settings/python3-returnn-tf1.9/lib/python3.5/site-packages/tensorflow/python/client/[0m[36;1msession.py[0m[36m"[0m, [34mline [0m[35m877[0m, [34min [0mrun
[34m    line: [0mresult [34m=[0m self[34m.[0m_run[34m([0m[34mNone[0m[34m,[0m fetches[34m,[0m feed_dict[34m,[0m options_ptr[34m,[0m
                             run_metadata_ptr[34m)[0m
[34m    locals:[0m
      result [34;1m= [0m[34m<not found>[0m
      self [34;1m= [0m[34m<local> [0m[34m<[0mtensorflow[34m.[0mpython[34m.[0mclient[34m.[0msession[34m.[0mSession object at 0x2b77e8e15cc0[34m>[0m
      self[34;1m.[0m_run [34;1m= [0m[34m<local> [0m[34m<[0mbound method BaseSession[34m.[0m_run of [34m<[0mtensorflow[34m.[0mpython[34m.[0mclient[34m.[0msession[34m.[0mSession object at 0x2b77e8e15cc0[34m>[0m[34m>[0m
      fetches [34;1m= [0m[34m<local> [0m[36m'saver/save/restore_all'[0m[34m,[0m len [34m=[0m 22
      feed_dict [34;1m= [0m[34m<local> [0m[34m{[0m[36m'saver/save/Const:0'[0m[34m:[0m [36m'/work/smt3/bahar/expriments/asr/librispeech/test/20190121-end2end-hmm/data-train/trafo.specaug.datarndperm_noscale.12l.ffdim4.pretrain3-prior-k6/net-model/network.560'[0m[34m}[0m
      options_ptr [34;1m= [0m[34m<local> [0m[34mNone[0m
      run_metadata_ptr [34;1m= [0m[34m<local> [0m[34mNone[0m
  [34;1mFile [0m[36m"/u/bahar/settings/python3-returnn-tf1.9/lib/python3.5/site-packages/tensorflow/python/client/[0m[36;1msession.py[0m[36m"[0m, [34mline [0m[35m1100[0m, [34min [0m_run
[34m    line: [0mresults [34m=[0m self[34m.[0m_do_run[34m([0mhandle[34m,[0m final_targets[34m,[0m final_fetches[34m,[0m
                                 feed_dict_tensor[34m,[0m options[34m,[0m run_metadata[34m)[0m
[34m    locals:[0m
      results [34;1m= [0m[34m<not found>[0m
      self [34;1m= [0m[34m<local> [0m[34m<[0mtensorflow[34m.[0mpython[34m.[0mclient[34m.[0msession[34m.[0mSession object at 0x2b77e8e15cc0[34m>[0m
      self[34;1m.[0m_do_run [34;1m= [0m[34m<local> [0m[34m<[0mbound method BaseSession[34m.[0m_do_run of [34m<[0mtensorflow[34m.[0mpython[34m.[0mclient[34m.[0msession[34m.[0mSession object at 0x2b77e8e15cc0[34m>[0m[34m>[0m
      handle [34;1m= [0m[34m<local> [0m[34mNone[0m
      final_targets [34;1m= [0m[34m<local> [0m[34m[[0m[34m<[0mtf[34m.[0mOperation [36m'saver/save/restore_all'[0m type[34m=[0mNoOp[34m>[0m[34m][0m
      final_fetches [34;1m= [0m[34m<local> [0m[34m[[0m[34m][0m
      feed_dict_tensor [34;1m= [0m[34m<local> [0m[34m{[0m[34m<[0mtf[34m.[0mTensor [36m'saver/save/Const:0'[0m shape[34m=[0m[34m([0m[34m)[0m dtype[34m=[0mstring[34m>[0m[34m:[0m array[34m([0m[36m'/work/smt3/bahar/expriments/asr/librispeech/test/20190121-end2end-hmm/data-train/trafo.specaug.datarndperm_noscale.12l.ffdim4.pretrain3-prior-k6/net-model/network.560'[0m[34m,[0m
                                       dtype[34m=[0mobject[34m)[0m[34m}[0m
      options [34;1m= [0m[34m<local> [0m[34mNone[0m
      run_metadata [34;1m= [0m[34m<local> [0m[34mNone[0m
  [34;1mFile [0m[36m"/u/bahar/settings/python3-returnn-tf1.9/lib/python3.5/site-packages/tensorflow/python/client/[0m[36;1msession.py[0m[36m"[0m, [34mline [0m[35m1272[0m, [34min [0m_do_run
[34m    line: [0m[34mreturn [0mself[34m.[0m_do_call[34m([0m_run_fn[34m,[0m feeds[34m,[0m fetches[34m,[0m targets[34m,[0m options[34m,[0m
                               run_metadata[34m)[0m
[34m    locals:[0m
      self [34;1m= [0m[34m<local> [0m[34m<[0mtensorflow[34m.[0mpython[34m.[0mclient[34m.[0msession[34m.[0mSession object at 0x2b77e8e15cc0[34m>[0m
      self[34;1m.[0m_do_call [34;1m= [0m[34m<local> [0m[34m<[0mbound method BaseSession[34m.[0m_do_call of [34m<[0mtensorflow[34m.[0mpython[34m.[0mclient[34m.[0msession[34m.[0mSession object at 0x2b77e8e15cc0[34m>[0m[34m>[0m
      _run_fn [34;1m= [0m[34m<local> [0m[34m<[0mfunction BaseSession[34m.[0m_do_run[34m.[0m[34m<[0mlocals[34m>[0m[34m.[0m_run_fn at 0x2b7c29e46b70[34m>[0m
      feeds [34;1m= [0m[34m<local> [0m[34m{[0m[34m<[0mtensorflow[34m.[0mpython[34m.[0mpywrap_tensorflow_internal[34m.[0mTF_Output[34m;[0m proxy of [34m<[0mSwig Object of type [36m'TF_Output *'[0m at 0x2b7842423120[34m>[0m [34m>[0m[34m:[0m array[34m([0m[36m'/work/smt3/bahar/expriments/asr/librispeech/test/20190121-end2end-hmm/data-train/trafo.specaug.datarndperm_noscale.12l.ffdim4.pretrain3-prior-k6/net-model/network.560...[0m
      fetches [34;1m= [0m[34m<local> [0m[34m[[0m[34m][0m
      targets [34;1m= [0m[34m<local> [0m[34m[[0m[34m<[0mSwig Object of type [36m'TF_Operation *'[0m at 0x2b7c2aac6960[34m>[0m[34m][0m
      options [34;1m= [0m[34m<local> [0m[34mNone[0m
      run_metadata [34;1m= [0m[34m<local> [0m[34mNone[0m
  [34;1mFile [0m[36m"/u/bahar/settings/python3-returnn-tf1.9/lib/python3.5/site-packages/tensorflow/python/client/[0m[36;1msession.py[0m[36m"[0m, [34mline [0m[35m1291[0m, [34min [0m_do_call
[34m    line: [0m[34mraise [0mtype[34m([0me[34m)[0m[34m([0mnode_def[34m,[0m op[34m,[0m message[34m)[0m
[34m    locals:[0m
      type [34;1m= [0m[34m<builtin> [0m[34m<[0m[34mclass [0m[36m'type'[0m[34m>[0m
      e [34;1m= [0m[34m<not found>[0m
      node_def [34;1m= [0m[34m<local> [0mname[34m:[0m [36m"saver/save/RestoreV2"[0m
                         op[34m:[0m [36m"RestoreV2"[0m
                         input[34m:[0m [36m"saver/save/Const"[0m
                         input[34m:[0m [36m"saver/save/RestoreV2/tensor_names"[0m
                         input[34m:[0m [36m"saver/save/RestoreV2/shape_and_slices"[0m
                         device[34m:[0m [36m"/device:CPU:0"[0m
                         attr [34m{[0m
                           key[34m:[0m [36m"dtypes"[0m
                           value [34m{[0m
                             list [34m{[0m
                               type[34m:[0m DT_FLOAT
                               type[34m:[0m DT_FLOAT
                               type[34m:[0m DT_FLOAT
                              [34m.[0m[34m.[0m[34m.[0m
      op [34;1m= [0m[34m<local> [0m[34m<[0mtf[34m.[0mOperation [36m'saver/save/RestoreV2'[0m type[34m=[0mRestoreV2[34m>[0m
      message [34;1m= [0m[34m<local> [0m[36m'/work/smt3/bahar/expriments/asr/librispeech/test/20190121-end2end-hmm/data-train/trafo.specaug.datarndperm_noscale.12l.ffdim4.pretrain3-prior-k6/net-model/network.560.index; Permission denied\n\t [[Node: saver/save/RestoreV2 = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ...., len = 850[0m
[31mPermissionDeniedError[0m: /work/smt3/bahar/expriments/asr/librispeech/test/20190121-end2end-hmm/data-train/trafo.specaug.datarndperm_noscale.12l.ffdim4.pretrain3-prior-k6/net-model/network.560.index; Permission denied
	 [[Node: saver/save/RestoreV2 = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_saver/save/Const_0_0, saver/save/RestoreV2/tensor_names, saver/save/RestoreV2/shape_and_slices)]]
	 [[Node: saver/save/RestoreV2/_181 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device_incarnation=1, tensor_name="edge_186_saver/save/RestoreV2", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"]()]]

Caused by op 'saver/save/RestoreV2', defined at:
  File "/u/makarov/returnn-parnia/tools/get-attention-weights.py", line 414, in <module>
    main(sys.argv)
  File "/u/makarov/returnn-parnia/tools/get-attention-weights.py", line 231, in main
    init_net(args, layers)
  File "/u/makarov/returnn-parnia/tools/get-attention-weights.py", line 139, in init_net
    rnn.engine.init_network_from_config(config=config, net_dict_post_proc=net_dict_post_proc)
  File "/u/makarov/returnn-parnia/TFEngine.py", line 967, in init_network_from_config
    self.network.load_params_from_file(model_epoch_filename, session=self.tf_session)
  File "/u/makarov/returnn-parnia/TFNetwork.py", line 947, in load_params_from_file
    self._create_saver()
  File "/u/makarov/returnn-parnia/TFNetwork.py", line 904, in _create_saver
    var_list=self.get_saveable_params_list(), max_to_keep=2 ** 31 - 1)
  File "/u/bahar/settings/python3-returnn-tf1.9/lib/python3.5/site-packages/tensorflow/python/training/saver.py", line 1281, in __init__
    self.build()
  File "/u/bahar/settings/python3-returnn-tf1.9/lib/python3.5/site-packages/tensorflow/python/training/saver.py", line 1293, in build
    self._build(self._filename, build_save=True, build_restore=True)
  File "/u/bahar/settings/python3-returnn-tf1.9/lib/python3.5/site-packages/tensorflow/python/training/saver.py", line 1330, in _build
    build_save=build_save, build_restore=build_restore)
  File "/u/bahar/settings/python3-returnn-tf1.9/lib/python3.5/site-packages/tensorflow/python/training/saver.py", line 778, in _build_internal
    restore_sequentially, reshape)
  File "/u/bahar/settings/python3-returnn-tf1.9/lib/python3.5/site-packages/tensorflow/python/training/saver.py", line 397, in _AddRestoreOps
    restore_sequentially)
  File "/u/bahar/settings/python3-returnn-tf1.9/lib/python3.5/site-packages/tensorflow/python/training/saver.py", line 829, in bulk_restore
    return io_ops.restore_v2(filename_tensor, names, slices, dtypes)
  File "/u/bahar/settings/python3-returnn-tf1.9/lib/python3.5/site-packages/tensorflow/python/ops/gen_io_ops.py", line 1463, in restore_v2
    shape_and_slices=shape_and_slices, dtypes=dtypes, name=name)
  File "/u/bahar/settings/python3-returnn-tf1.9/lib/python3.5/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/u/bahar/settings/python3-returnn-tf1.9/lib/python3.5/site-packages/tensorflow/python/util/deprecation.py", line 454, in new_func
    return func(*args, **kwargs)
  File "/u/bahar/settings/python3-returnn-tf1.9/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 3155, in create_op
    op_def=op_def)
  File "/u/bahar/settings/python3-returnn-tf1.9/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 1717, in __init__
    self._traceback = tf_stack.extract_stack()

PermissionDeniedError (see above for traceback): /work/smt3/bahar/expriments/asr/librispeech/test/20190121-end2end-hmm/data-train/trafo.specaug.datarndperm_noscale.12l.ffdim4.pretrain3-prior-k6/net-model/network.560.index; Permission denied
	 [[Node: saver/save/RestoreV2 = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_saver/save/Const_0_0, saver/save/RestoreV2/tensor_names, saver/save/RestoreV2/shape_and_slices)]]
	 [[Node: saver/save/RestoreV2/_181 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device_incarnation=1, tensor_name="edge_186_saver/save/RestoreV2", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"]()]]

2019-07-02 12:24:21.394418: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2019-07-02 12:24:22.514853: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1405] Found device 0 with properties: 
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.582
pciBusID: 0000:03:00.0
totalMemory: 10.92GiB freeMemory: 10.76GiB
2019-07-02 12:24:22.514899: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
2019-07-02 12:24:22.514913: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-07-02 12:24:22.514920: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
2019-07-02 12:24:22.514926: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
2019-07-02 12:24:22.517592: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
2019-07-02 12:24:22.952105: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-07-02 12:24:22.952158: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
2019-07-02 12:24:22.952166: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
2019-07-02 12:24:22.952543: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/device:GPU:0 with 10405 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:03:00.0, compute capability: 6.1)
ERROR: cannot read default config file. using build-in defaults [main]
LOG: connected to ('10.6.100.1', 10321)
LOG: destination: /var/tmp/makarov/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: using existing file
2019-07-02 12:24:24.238695: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
2019-07-02 12:24:24.238750: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-07-02 12:24:24.238759: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
2019-07-02 12:24:24.238765: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
2019-07-02 12:24:24.238932: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10405 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:03:00.0, compute capability: 6.1)
/var/spool/sge/cluster-cn-244/job_scripts/9506015: line 8: 28051 Killed                  python3 ~/returnn-parnia/tools/get-attention-weights.py /u/bahar/workspace/asr/librispeech/test-20190121/config-train/trafo.specaug.datarndperm_noscale.12l.ffdim4.pretrain3-prior-k6.config --epoch 560 --data 'config:get_dataset("'$1'")' --dump_dir ~/makarov/NMT/hmm-factorization/experiments/asr2019/transformer-prior/forward-$1 --layers "dec_02_att_weights" --layers "dec_01_att_weights" --layers "dec_03_att_weights" --layers "dec_04_att_weights" --layers "dec_05_att_weights" --layers "dec_06_att_weights" --layers "dec_07_att_weights" --layers "dec_08_att_weights" --layers "dec_09_att_weights" --layers "dec_10_att_weights" --layers "dec_11_att_weights" --layers "dec_12_att_weights" --rec_layer "output" --batch_size 4000 --tf_log_dir /work/smt2/makarov/NMT/hmm-factorization/experiments/asr2019/transformer-prior/forward-$1/tf_log_dir --reset_seq_ordering sorted_reverse
2019-07-02 12:35:24.338157: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2019-07-02 12:35:25.366649: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1405] Found device 0 with properties: 
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.582
pciBusID: 0000:81:00.0
totalMemory: 10.92GiB freeMemory: 10.76GiB
2019-07-02 12:35:25.366951: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
2019-07-02 12:35:25.366990: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-07-02 12:35:25.366999: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
2019-07-02 12:35:25.367005: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
2019-07-02 12:35:25.370598: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
2019-07-02 12:35:25.789299: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-07-02 12:35:25.789350: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
2019-07-02 12:35:25.789356: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
2019-07-02 12:35:25.789660: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/device:GPU:0 with 10405 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:81:00.0, compute capability: 6.1)
ERROR: cannot read default config file. using build-in defaults [main]
LOG: connected to ('10.6.100.1', 10321)
LOG: destination: /var/tmp/makarov/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: request: /work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-221:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-221:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-215:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: cannot get stat of cluster-cn-215:/var/tmp/irie/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: checking status of cluster-cn-02:/var/tmp/zeineldeen/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: start copying cluster-cn-02:/var/tmp/zeineldeen/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
LOG: copied cluster-cn-02:/var/tmp/zeineldeen/work/asr3/zeyer/setups-data/librispeech/dataset/ogg-zips/dev-clean.zip
2019-07-02 12:35:31.844533: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
2019-07-02 12:35:31.844612: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-07-02 12:35:31.844625: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
2019-07-02 12:35:31.844636: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
2019-07-02 12:35:31.844879: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10405 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:81:00.0, compute capability: 6.1)
